### ISSUE TITLE
name local

### ISSUE THREAD

**Issue by thedreamteam2** [ProjectEssentials/ProjectEssentials#12]
hello

i hope somebody can help we are using projectessentials ,,problem is when we make a group and add ourself to that group,,and we typ in the chat its still saying local in front of the name and not the name of the group,, is there a way to change that?

**Comment by hujle** [comment]
local is not a group. check [https://mairwunnx.gitbook.io/project-essentials/project-essentials-chat](PE-Chat configuration page).

**Comment by thedreamteam2** [comment]
yeah we know that the problem we have is that we want to have owner in front of the name

**Comment by hujle** [comment]
it is not possible in current state because that function is not exists in that mod so.

**Comment by thedreamteam2** [comment]
okay to bad it would be nice if we could do that thanks for youre answer

**Comment by MairwunNx** [comment]
@thedreamteam2 yes, currently this feature not implemented. I will implement this in next major version. Thanks 🍀.
